experimental/devices/ads1x15/ads1x15.go

```diff
@@ -0,0 +1,392 @@
+package ads1x15
+
+import (
+	"encoding/binary"
+	"errors"
+	"math"
+	"sync"
+	"time"
+
+	"periph.io/x/periph/conn/physic"
+	"periph.io/x/periph/conn/pin"
+
+	"fmt"
+
+	"periph.io/x/periph/conn/i2c"
+)
+
+const (
+
+	// ADS1x15DefaultAddress is the default I2C address for the ADS1x15 components
+	ADS1x15DefaultAddress = 0x48
```

Comment:
It's fine to use I2CAddr, so that at usage it looks like ads1x15.I2CAddr. Right now it stutters.

Reply:
Done